contracts/AtlasMineStakerUpgradeable.sol

```diff
@@ -534,9 +521,6 @@ contract AtlasMineStakerUpgradeable is
      *         all stake.
      */
     function unstakeAllFromMine() external override onlyOwner {
-        // Unstake everything eligible
-        _updateRewards();
-
```

Comment:
practically we wont be using this i think, and maybe duplicative with `emergencyUnstakeAllFromMine`?

but again i get if we want to remove for clarity's sake